src/main/java/com/github/namikon/angermod/events/BlockBreakEvent.java

```diff
@@ -52,4 +50,65 @@ else if (pEvent.getPlayer().dimension == 1) // End
             AngerMod.Logger.DumpStack("BlockBreakEvent.onBreakBlock.Stack", e);
         }
     }
+
+    @SuppressWarnings("unchecked")
+    public static void aggroEndermenInRange(EntityPlayer player, int range) {
+        List nearbyEntities;
+
+        int x = (int) player.posX;
+        int y = (int) player.posY;
+        int z = (int) player.posZ;
+
+        // Define the area to check for entities
+        AxisAlignedBB tBoundingBox = AxisAlignedBB
+                .getBoundingBox(x - range, y - range, z - range, x + range, y + range, z + range);
+
+        nearbyEntities = player.worldObj.getEntitiesWithinAABB(EntityEnderman.class, tBoundingBox);
+
+        for (Entity entity : (List<Entity>) nearbyEntities) {
+            if (entity instanceof EntityEnderman enderman) {
+                enderman.setTarget(player);
+                enderman.setScreaming(true);
+            }
+        }
+    }
+
+    private static Method becomeAngryAtMethod;
+
+    static {
+        try {
+            becomeAngryAtMethod = EntityPigZombie.class.getDeclaredMethod("becomeAngryAt", Entity.class);
```

Comment:
This will not work outside of dev, you need to check the SRG name too (func_12345 etc.)